app/models/item.rb

```diff
@@ -15,12 +16,11 @@ class Item < ApplicationRecord
   belongs_to :user
   has_and_belongs_to_many :allergens
   has_and_belongs_to_many :diets
-  validates_presence_of :user_id, :description, :expiration_date, :type, :status
+  validates_presence_of :user_id, :description, :expiration_date, :type, :status, :name
```

Comment:
I changed type to item type in my branch, but if you do it as well here, there won't be any merge conflicts when I push my seed branch.

Reply:
Updated the field name in the Items table!